docs/_pages/0980-release-management.md

```diff
@@ -249,7 +251,19 @@ now can merge our changes to the `main` branch.
 Create a PR from the release branch you are on, and follow the regular merge
 approval process.
 
-### 9. Release helm-charts
+### 9. Tag the `main` Branch
+
+We have created a release in the previous step, but we did it on a feature 
+branch. Now, we need to tag the `main` branch with the same version.
+
+Here’s an example:
+
+```bash
+git tag -s v0.22.0-000 -a v0.22.0-000
+git push origin --tags
+```
```

Comment:
This is needed to see what commit hash on the main branch correlates to a release cut. 

Otherwise, because of the way we cut releases, we cannot see the tags on the main branch.